.github/workflows/integration.yml

```diff
@@ -49,8 +91,22 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [netcoreapp3.1,net5.0]
+        version: [netcoreapp3.1,net5.0,net6.0]
     steps:
       - uses: actions/checkout@v2
+
+      - uses: actions/setup-dotnet@v1
```

Comment:
same here.
you can check this:
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md